ikasaneip/builder/jar/src/test/java/org/ikasan/builder/component/endpoint/DbConsumerBuilderTest.java

```diff
@@ -140,6 +141,8 @@ public void dbConsumer_build_when_all_attributes_set_scheduledConsumer_vanilla_m
                 .setSqlStatement("sqlStatement")
                 .build();
 
+        ScheduledConsumer scheduledConsumer = (ScheduledConsumer)consumer;
```

Comment:
Pedantic point; this:
`...(ScheduledConsumer)consumer`
Should be this:
`...(ScheduledConsumer) consumer`
>

Reply:
Unless that's not our pattern?